### ISSUE TITLE
extension still necessary since site sets are available in TYPO3?

### ISSUE THREAD

**Issue by josefglatz** [b13/bolt#48]
I'm new to site sets. I love b13/bolt. I'm actually thinking, that bolt isn't adding any advantage on top if you switch you project to site sets?

**Comment by bmack** [comment]
That is correct... You know what my reasoning is all the time: Create an extension (this time it was v7) and at some point it becomes obsolete because Core has something better....

I think we are going to keep a compatible version at least for v13 in order to make the migration easier.

**Comment by josefglatz** [comment]
It was a great time Benni! TYPO3 websites without sys_templates records without any 3rd party package: next level awesomeness!!! The whole site sets feature and PAGEVIEW cObject is a really big enhancement in this area!
